dash/frontend/src/app/modules/private/private.component.ts

```diff
@@ -217,7 +217,6 @@ export class PrivateComponent implements OnInit, AfterViewInit, OnDestroy {
       data: {}
     });
     confirmDialog.afterClosed().subscribe(result => {
-      this.directive.ngOnInit();
```

Comment:
What purpose did this serve? I'm wondering if this might not be required somewhere.
Thoughts @kyle-intelletive?

Reply:
I can take a look

Reply:
It looks like directive is a ViewChild reference to a UserProfileImageDirective. My assumption is that this code used to cause that to reload the user profile picture.

Since we don't atually allow that, we probably don't need anything related to that ViewChild, or this subscription to the dialogref.afterclosed()

Reply:
In that case, should we add that to the list of this to tear out?

Reply:
Yes, while the directive is still actually being used, as far as I can tell it is entirely useless, since all it has is a constructor with nothing in it

Reply:
@kyle-intelletive After removing that line of code, name change was updated. Do you want me to leave this as is for now?

Reply:
Yeah don't worry about it, if we remove the directive we can remove the rest at the point.